src/main.py

```diff
@@ -63,7 +63,17 @@
 ASSISTANT_CREDENTIALS = os.path.join(VR_CACHE_DIR, 'assistant_credentials.json')
 ASSISTANT_OAUTH_SCOPE = 'https://www.googleapis.com/auth/assistant-sdk-prototype'
 
-PID_FILE = '/run/user/%d/voice-recognizer.pid' % os.getuid()
+# Grab pid file info
+USER_ID = '%d' % os.getuid()
+GROUP_ID = '%d' % os.getgid()
+PID_PATH = '/run/user/' + USER_ID
+PID_FILE = PID_PATH + '/voice-recognizer.pid'
+PID_COMMAND = 'sudo mkdir ' + PID_PATH + ' && sudo chown -R ' + USER_ID + ':' + GROUP_ID + ' ' + PID_PATH
+
+
+# Create and own pid directory if it doesn't exist
+if not os.path.exists(PID_PATH):
+    os.system(PID_COMMAND)
```

Comment:
Could you please squash rebase the PR so that the reverts are gone. It seems that
"Add Rpi.GPIO to pip dependencies." is gone from your changes - is it not needed anymore? On raspbian this is pre-installed as ```python3-rpi.gpio```. It would be better to add it to the apt-get install part to make sure this part is neutral for raspbian users.